### PR TITLE
docs: Add Husky.Net option

### PR DESCRIPTION
**Pull request by alirezanet**
## Description

Dotnet developers should use a dotnet-tool solution along with prettier

## Checklist

- [x] I’ve added tests to confirm my change works.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Review by sosukesuzuki — APPROVED** [review]
I don't use `dotnet` so I'm not sure this is correct..

File: docs/precommit.md
Comment (fisker) [inline docs/precommit.md:79]: We suggest use `--ignore-unknown` flag and include all files.

Comment (alirezanet) [reply docs/precommit.md]: good suggestion thanks. updated.

**Comment by fisker** [comment]
Cann you copy this part to https://github.com/prettier/prettier/blob/main/website/versioned_docs/version-stable/precommit.md?

**Comment by alirezanet** [comment]
> Can you copy this part to https://github.com/prettier/prettier/blob/main/website/versioned_docs/version-stable/precommit.md?

Another PR?

**Comment by fisker** [comment]
> Another PR?

No, should done in this PR, the website use the file I linked, update it will make prettier.io update the doc, no need wait for next release.

**Comment by alirezanet** [comment]
> No, should done in this PR, the website use the file I linked, update it will make prettier.io update the doc, no need wait for next release.

You’re making changes in a project you don’t have write access to. Submitting a change will write it to a new branch in your fork alirezanet/prettier, so you can send a pull request.

**Comment by fisker** [comment]
Don't worry, I'll fix.

**Review by fisker — APPROVED** [review]

**Comment by fisker** [comment]
Thanks for contributing.

https://prettier.io/docs/en/precommit.html#option-4-huskynethttpsgithubcomalirezanethuskynet

**Pull request merged**
